alembic/templates/generic/script.py.mako

```diff
@@ -5,8 +5,9 @@ Revises: ${down_revision | comma,n}
 Create Date: ${create_date}
 
 """
-from alembic import op
```

Comment:
this is fine but would need to be applied to all the script.py.mako templates in all the configs.

Reply:
Done.